src/controllers/users.controller.ts

```diff
@@ -31,6 +32,53 @@ export const UserController = new Elysia({ prefix: "/users" })
 			return await userService.getUserByEmail(id);
 		},
 	)
+
+	//getAllProjectOwnerByProjectId
+	.get(
+		"/projectowner/:id",
+		async ({
+			params: { id },
+			db,
+			redis,
+		}: Context & { params: { id: string } }) => {
+			const userService = new UserService(db, redis);
+			try {
+				const user = await userService.getUserOwnersByProjectId(id);
+				return {
+					status: 200,
+					data: user,
+				};
+			} catch (error) {
+				return {
+					status: error instanceof NotFoundException ? 404 + "error" : 500,
+				};
+			}
+		},
+	)
```

Comment:
dont have to run try catch on controller just throw error only service file and middleware will handle exception